<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditProject</class>
 <widget class="QWidget" name="EditProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color:rgb(251, 245, 221);
    font-family: 'Segoe UI';
    font-size: 18px;
    color: #333;
}</string>
  </property>
  <widget class="QToolButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>üîç</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-weight: bold;
    color:rgb(61, 141, 122);
}</string>
   </property>
   <property name="text">
    <string> üîç Search Project : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSearch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid #A0A0A0;
    padding: 4px;
    border-radius: 5px;
    background-color: #FFFFFF;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-weight: bold;
    color:rgb(111, 130, 106);
}</string>
   </property>
   <property name="text">
    <string>Client  :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-weight: bold;
    color:rgb(111, 130, 106);
}</string>
   </property>
   <property name="text">
    <string>Project Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProjectLine">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>140</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid #A0A0A0;
    padding: 4px;
    border-radius: 5px;
    background-color: #FFFFFF;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-weight: bold;
    color:rgb(111, 130, 106);
}</string>
   </property>
   <property name="text">
    <string>Start Date  :</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="StartDateLine">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
border: 1px solid #A0A0A0;
    padding: 4px;
    border-radius: 5px;
    background-color: #FFFFFF;
}</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-weight: bold;
    color:rgb(111, 130, 106);
}</string>
   </property>
   <property name="text">
    <string>End Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="EndDateLine">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
border: 1px solid #A0A0A0;
    padding: 4px;
    border-radius: 5px;
    background-color: #FFFFFF;
}</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="BudgetBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
  background-color: white;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
    font-size: 13px;
}</string>
   </property>
   <property name="prefix">
    <string>$</string>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-weight: bold;
    color:rgb(111, 130, 106);
}</string>
   </property>
   <property name="text">
    <string>Budget :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="StatusBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
border: 1px solid #A0A0A0;
    padding: 4px;
    border-radius: 5px;
    background-color: #FFFFFF;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>245</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-weight: bold;
    color:rgb(111, 130, 106);
}</string>
   </property>
   <property name="text">
    <string>Status :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-weight: bold;
    color:rgb(111, 130, 106);
}</string>
   </property>
   <property name="text">
    <string>Description :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="DescriptionText">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>471</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
border: 1px solid #A0A0A0;
    padding: 4px;
    border-radius: 5px;
    background-color: #FFFFFF;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnUpdate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>500</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color:#6F826A;
    color: white;
    padding: 6px;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #BBD8A3;
}</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCnacel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color:#6F826A;
    color: white;
    padding: 6px;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #BBD8A3;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnDelete">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>500</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color:#6F826A;
    color: white;
    padding: 6px;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #BBD8A3;
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QToolButton" name="BtnBack">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
   color: white;
    background-color:#6F826A;
    border-radius: 6px;
    font-weight: bold;
}
QToolButton:hover{
  background-color:#BBD8A3;
}</string>
   </property>
   <property name="text">
    <string>&lt;--</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ClientLine">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid #A0A0A0;
    padding: 4px;
    border-radius: 5px;
    background-color: #FFFFFF;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
